include/dxc/DxilContainer/DxilPipelineStateValidation.h

```diff
@@ -1078,6 +1086,8 @@ DxilPipelineStateValidation::ReadOrWrite(const void *pBits, uint32_t *pSize,
   if (mode == RWMode::CalcSize) {
     *pSize = rw.GetSize();
     m_pPSVRuntimeInfo1 = nullptr; // clear ptr to tempRuntimeInfo
+  } else if (mode == RWMode::Read) {
+    *pSize = rw.GetOffset();
```

Comment:
We should not need this, as the passed in size when reading is the size of the container part.
CalcSize mode should be sufficient for validating the container part size is that calculated size, aligned, and any padding for alignment is filled with zero.

Reply:
Reverting this eliminates the need to add `GetOffset()` as well.